tasks/views.py

```diff
@@ -1,6 +1,21 @@
+from time import strftime
 from django.http import HttpResponse
 from django.shortcuts import render
+from .models import Task
 
 # Create your views here.
+# def list_tasks(request):
+#     data=Task.objects.all()
+#     return HttpResponse(data)
+
 def list_tasks(request):
-    return HttpResponse("List my task")
\ No newline at end of file
+    tasks = Task.objects.all()  # or filter(user=request.user) if user-specific
```

Comment:
```suggestion
    tasks = Task.objects.all()
```
Avoid commented code